src/main/resources/templates/rooms/checkIn.html

```diff
@@ -109,10 +109,6 @@ <h3 class="content-header form-header">Immunitätsnachweise</h3>
             </div>
 
             <form action="#" th:action="${checkInOverwrite} ? @{/r/checkInOverride} : @{/r/checkIn}" method="POST" id="submit-form" name="submit-form">
-                <h3 class="content-header form-header">Raum-PIN eingeben</h3>
```

Comment:
Maybe the consensus has changed, but I remember the PIN input field on this page being useful for people that access the room check-in via the URL printed above the QR codes.

Reply:
Thanks alot! Forgot about that possiblility to checkin completely, will ask in the next jour fix how we want to handle this situation